### ISSUE TITLE
Duplicate navigation landmarks need labels

### ISSUE THREAD

**Issue by AutoSponge** [hankchizljaw/hylia#64]
https://www.w3.org/TR/wai-aria-practices/examples/landmarks/navigation.html

The template's default is 2 `<nav>` elements, one in the header and another in the footer. They have identical content. But, this pattern violates the SC.

Suggestions:
- remove the footer nav. (label becomes optional).
- clone the template and add labels to each. `header_nav` and `footer_nav` (although naming them becomes somewhat troublesome, at least for me). I didn't see a way to pass data to the templates so this might need to be a short_code instead (seems hard).
- remove the `<nav>` element from the partial and only wrap the header version with it. The footer will just have a list of links. Still helpful, but not a (duplicate) landmark.

**Comment by andybelldesign** [comment]
Good catch! I think we’ll go with an `aria-label` on each instance.

**Comment by hankchizljaw** [comment]
Resolved with #102
